test/test_backbone_utils.py

```diff
@@ -23,6 +23,13 @@ def test_resnet_fpn_backbone(backbone_name):
     y = resnet_fpn_backbone(backbone_name=backbone_name, pretrained=False)(x)
     assert list(y.keys()) == ["0", "1", "2", "3", "pool"]
 
+    with pytest.raises(ValueError):
```

Comment:
We should also check for error message, kindly append that too.
E.g. https://github.com/pytorch/vision/blob/main/test/test_utils.py#L306

Reply:
oh, okay! Will do it